ChiModules/KEigenvalueSolver/k_eigenvalue_solver.h

```diff
@@ -16,7 +16,8 @@ class Solver : public LinearBoltzmann::Solver
   size_t source_event_tag;
 
 public:
```

Comment:
Please document these data members, especially the interpretation/indexing of ```precursor_map```. The text in the description of the pull request is clear, but it is less accessible than if it were directly in the source code.

Reply:
Done.